backend/src/routes/preferences/wishlist.router.ts

```diff
@@ -0,0 +1,15 @@
+import * as wishlistController from '@controllers/preferences/wishlist.controller';
+import { Router } from 'express';
+
+const PATH = '/wishlist';
+
+const wishlistRouter = Router();
+
+wishlistRouter.post(`${PATH}/model/:id`, wishlistController.setModelWishlist);
+
+wishlistRouter.post(
+  `${PATH}/complectation/:id`,
+  wishlistController.setComplectationWishlist,
+);
```

Comment:
These routes can be **simplified in one simple change**. Plus, there will be no duplication of the code in the endpoints. Actually, there will be **only one endpoint**.

You need to use a bit different route pattern: 
```suggestion
const wishlistRouter = Router();

wishlistRouter.post(
  `${PATH}/wishlist/:userId`,
  wishlistController.setWishlist,
);
```
And then you can provide either the `?modelId=` or the `?complectationId=` as the URL params.
Then, in the `setWishlist` you can do smth like:
```ts
    ...
    const { userId } = req.body;
    const { modelId, complectationId } = req.params;

    const wishlist = {
      userId,
      modelId,
      complectationId,
    };

    ...
```

This way, the whole wishlist logic takes much less lines of code and adding the new wishlist type would be very easy